core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Client.java

```diff
@@ -22,8 +22,6 @@
 
 public interface OAuth2Client
 {
-    void load();
```

Comment:
This method do not fit into this interface as it has to be loaded only once during initialization. So any client of that instance should not be able to call it again.

Reply:
Why not have a factory or just @Provider in guice module?

Reply:
If you bind as SINGLETON I think it would be called only once.